load-tests/h1c_transformation/src/h1c_transformation.bal

```diff
@@ -18,39 +18,39 @@ import ballerina/http;
 import ballerina/log;
 import ballerina/xmldata;
 
-final http:Client nettyEP = check new("http://netty:8688");
+final http:Client nettyEP = check new("http://netty:8688", httpVersion = "1.1");
 
-service /transform on new http:Listener(9090) {
-    resource function post .(http:Caller caller, http:Request req) {
+service /transform on new http:Listener(9090, httpVersion = "1.1") {
+    resource function post .(http:Caller caller, http:Request req) returns error? {
         json|error payload = req.getJsonPayload();
-        if (payload is json) {
+        if payload is json {
             xml|xmldata:Error? xmlPayload = xmldata:fromJson(payload);
-            if (xmlPayload is xml) {
+            if xmlPayload is xml {
                 http:Request clinetreq = new;
                 clinetreq.setXmlPayload(xmlPayload);
                 http:Response|http:ClientError response = nettyEP->post("/service/EchoService", clinetreq);
                 if (response is http:Response) {
-                    error? result = caller->respond(response);
+                    return caller->respond(response);
                 } else {
                     log:printError("Error at h1c_transformation", 'error = response);
                     http:Response res = new;
                     res.statusCode = 500;
                     res.setPayload(response.message());
-                    error? result = caller->respond(res);
+                    return caller->respond(res);
                 }
-            } else if (xmlPayload is xmldata:Error) {
+            } else if xmlPayload is xmldata:Error {
                 log:printError("Error at h1c_transformation", 'error = xmlPayload);
                 http:Response res = new;
                 res.statusCode = 400;
                 res.setPayload(xmlPayload.message());
-                error? result = caller->respond(res);
+                return caller->respond(res);
             }
         } else {
             log:printError("Error at h1c_transformation", 'error = payload);
             http:Response res = new;
             res.statusCode = 400;
             res.setPayload(payload.message());
-            error? result = caller->respond(res);
+            return caller->respond(res);
```

Comment:
Here also, simply return http:Response

Reply:
Fixed with [03ab7a3739f257aa4ac62770af56a7b3eff4804e](https://github.com/ballerina-platform/module-ballerina-http/pull/1166/commits/03ab7a3739f257aa4ac62770af56a7b3eff4804e) and [3aeae3bb7dcddc0e29f8675c34b21ba3ea3e9684](https://github.com/ballerina-platform/module-ballerina-http/pull/1166/commits/3aeae3bb7dcddc0e29f8675c34b21ba3ea3e9684)